README.md

```diff
@@ -46,21 +46,33 @@ features = ["libm"] # Makes it use libm instead of std for float math
+
+The informal "RGB" color space is such a tristimulus color space, but it is underspecified since we don't know which primaries are being used (i.e. how exactly are the canonical "red", "green", and "blue" defined?). In most cases, when people talk about "RGB" or "Linear RGB" colors, what they are *actually* talking about is "Linear sRGB" color space, which uses the primaries and white point defined in the sRGB standard, but which *does not* have the non-linear transfer function applied.
+
+This library takes these things into account, and attempts to provide an interface which will let those who don't care so much about the intricacies of color still use colors correctly, while also allowing the advanced user a high degree of flexibility in how they use it.
 
 ## What It Can Do
 
 Palette provides tools for both color manipulation and conversion between color spaces. These are some highlights.
 
 ### Color Spaces
 
-RGB is probably the most widely known color space, but it's not the only one. You have probably used a color picker with a rainbow wheel and a brightness slider. That may have been an HSV or an HSL color picker, where the color is encoded as hue, saturation and brightness/lightness. There's also a group of color spaces that are designed to be perceptually uniform, meaning that the perceptual change is equal to the numerical change.
+"RGB" (which we now know is usually Linear sRGB) and other tristimulus based spaces like CIE Xyz are probably the most widely known color spaces, and is great when you want to perform physically correct math on color, like in a 2d or 3d rendering program, but there are also color spaces that are not defined in terms of tristimulus values.
+
+You have probably used a color picker with a rainbow wheel and a brightness slider. That may have been an HSV or an HSL color picker, where the color is encoded as hue, saturation and brightness/lightness. Even though these spaces are defined using 3 values, they *aren't* based on tristimulus values, since those three values don't have a direct relation to human vision (i.e. our S, M, and L cones). Such color spaces are excellent when it comes to humans intuitively selecting color values, though, and as such are the go-to choice when this interaction is needed. They can then be converted into other color spaces in order to actually perform modifications to them
+
+There's also a group of color spaces that are designed to be perceptually uniform, meaning that the perceptual change is equal to the numerical change. An example of this is the CIE L\*a\*b coor space. These color spaces are excellent when you want to "blend" between colors in a *perceptually pleasing* manner (for example, in a data visualization) rather than a *physically correct* one.
```

Comment:
Typo

```suggestion
There's also a group of color spaces that are designed to be perceptually uniform, meaning that the perceptual change is equal to the numerical change. An example of this is the CIE L\*a\*b color space. These color spaces are excellent when you want to "blend" between colors in a *perceptually pleasing* manner (for example, in a data visualization) rather than a *physically correct* one.
```